### ISSUE TITLE
link_nvidia_host_libraries.sh uses curl that is not part of the compat layer

### ISSUE THREAD

**Issue by akesandgren** [EESSI/software-layer#471]
link_nvidia_host_libraries.sh is using curl to grab nvliblist.conf from Apptainer, but curl is not part of the compat layer.

**Comment by ocaisa** [comment]
It is:
```
/cvmfs/software.eessi.io/versions/2023.06/compat/linux/x86_64/usr/bin/curl
```
Did you run into a specific issue?

**Comment by ocaisa** [comment]
In general, I was thinking of removing this step and just shipping `nvliblist.conf` if we are allowed...

**Comment by akesandgren** [comment]
Yeah, looks like I need to setup my container with a bit more care :-)
I lost the PATH into /cvmfs/software.eessi.io/...

**Comment by akesandgren** [comment]
So ignore
